Makefile

```diff
@@ -209,6 +209,38 @@ uptest: $(UPTEST) $(KUBECTL) $(KUTTL)
 	@KUBECTL=$(KUBECTL) KUTTL=$(KUTTL) CROSSPLANE_NAMESPACE=$(CROSSPLANE_NAMESPACE) $(UPTEST) e2e "${UPTEST_EXAMPLE_LIST}" --data-source="${UPTEST_DATASOURCE_PATH}" --setup-script=cluster/test/setup.sh --default-conditions="Test" || $(FAIL)
 	@$(OK) running automated tests
 
+# This target triggers an e2e test for testing provider configs.
+# It first builds and publishes the
+# provider-family-aws, provider-aws-ec2 and provider-aws-rds.
+# Then triggers the e2e provider config tests via `make`,
+# which resides in the `e2e/providerconfig-aws-e2e-test` directory
```

Comment:
This seems like a name that's longer than necessary to convey the information. How about just `e2e/providerconfig`

Reply:
I am fine with the renaming the folder. My reasoning was, since this directory is also an XP configuration package, I just named it the same with the configuration package name.

The reason for this long name for the configuration package is that we could have other testing configuration packages for other providers as well.